assets/src/edit-story/components/panels/test/sizePosition.js

```diff
@@ -315,5 +350,32 @@ describe('Panels/SizePosition', () => {
         width: dataPixels(160 * (200 / 120)),
       });
     });
+
+    it('should update lock ratio to false for default elements', () => {
+      const { getByTestId, pushUpdate } = renderSizePosition([
+        image,
+        imageWithSameSize,
+      ]);
+      fireEvent.click(getByTestId('lockRatio'));
+      expect(pushUpdate).toHaveBeenCalledWith({ lockAspectRatio: false });
+    });
+
+    it('should update lock ratio to false for default element and unlock element', () => {
+      const { getByTestId, pushUpdate } = renderSizePosition([
+        unlockImage,
+        imageWithSameSize,
+      ]);
+      fireEvent.click(getByTestId('lockRatio'));
+      expect(pushUpdate).toHaveBeenCalledWith({ lockAspectRatio: false });
+    });
+
+    it('should update lock ratio to true for unlock aspect ratio elements', () => {
+      const { getByTestId, pushUpdate } = renderSizePosition([
+        unlockImageWithSameSize,
+        unlockImage,
+      ]);
+      fireEvent.click(getByTestId('lockRatio'));
+      expect(pushUpdate).toHaveBeenCalledWith({ lockAspectRatio: true });
+    });
```

Comment:
Great tests, but the descriptions are a bit complex. They could be named much simpler (don't refer to your variables, just refer to what actually happens):

 * Should disable aspect ratio lock if all elements had lock enabled.
 * Should disable aspect ratio lock if elements had different settings for aspect ratio lock.
 * Should enable aspect ratio lock only if all elements had lock disabled.

Reply:
I think enabled or disabled are not correct options. It sounds like that ratio lock disabled to use?

Reply:
Updated. But disabled sounds like the ratio lock toggle disabled to use. Maybe use turn on or off instead?